samples/Chat/src/app/ChatScreen.tsx

```diff
@@ -43,28 +43,28 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
         const removedParticipantIds = listener.participantsRemoved.map(
           (p) => (p.id as CommunicationUserIdentifier).communicationUserId
         );
-        if (removedParticipantIds.includes(userId)) {
+        if (removedParticipantIds.includes(userIdentifier)) {
           const removedBy = (listener.removedBy.id as CommunicationUserIdentifier).communicationUserId;
-          endChatHandler(removedBy !== userId);
+          endChatHandler(removedBy !== userIdentifier);
         }
       });
       adapter.on('error', (e) => {
         console.error(e);
       });
       return adapter;
     },
-    [endChatHandler, userId]
+    [endChatHandler, userIdentifier]
   );
 
   const adapterArgs = useMemo(
     () => ({
       endpoint: endpointUrl,
-      userId: fromFlatCommunicationIdentifier(userId) as CommunicationUserIdentifier,
+      userIdentifier: fromFlatCommunicationIdentifier(userIdentifier) as CommunicationUserIdentifier,
```

Comment:
I think part of the confusion from reading the original bug is that we had `userId` as a string and `userId` as a CommunicationUserIdentifier.
I think anywhere a string a is used, userId is fine.
Anywhere `CommunicationIdentifier` is used it should be `userIdentifier` or `communicationUserIdentifier` (as userIdentifier just is userId expanded)

Reply:
Alternatively anywhere the string is used could be `identity` or `userIdentity` as then it more closely matches the azure portal:

![image](https://user-images.githubusercontent.com/2684369/164516459-3d022612-b46e-4722-9668-17e7abbdb665.png)

Reply:
to resurface this are we wanting to actually rename this? its breaking change and by the logic of the comment above to rename the `CommunicationIdentifierKind` types we need to change the names on the `AzureCommunicationsChatAdapter` which is a major change.

Reply:
This comment may not be left in the correct space, but renaming the storybook props is not a breaking change -- renaming an arguments/props into the composite interface is a breaking change.
My understanding of what we can achieve without a breaking change is simply ensure in storybook we never have two variables with the same name being two different types.